tests/src/performance/Scenario/JitBench/JitBenchHarness.cs

```diff
@@ -22,6 +23,62 @@ static void Main(string[] args)
         {
             var options = JitBenchHarnessOptions.Parse(args);
 
+            SetupStatics(options);
+
+            using (var h = new XunitPerformanceHarness(args))
+            {
+                ProcessStartInfo startInfo = options.UseExistingSetup ? UseExistingSetup() : CreateNewSetup();
+
+                string scenarioName = "MusicStore";
+
+                if (options.EnableTiering)
+                {
+                    startInfo.Environment.Add("COMPlus_EXPERIMENTAL_TieredCompilation", "1");
+                    scenarioName += " Tiering";
+                }
+
+                if (options.Minopts)
+                {
+                    startInfo.Environment.Add("COMPlus_JITMinOpts", "1");
+                    scenarioName += " Minopts";
+                }
+
+                if (options.DisableR2R)
+                {
+                    startInfo.Environment.Add("COMPlus_ReadyToRun", "0");
+                    scenarioName += " NoR2R";
+                }
+
+                if (options.DisableNgen)
+                {
+                    startInfo.Environment.Add("COMPlus_ZapDisable", "1");
+                    scenarioName += " NoNgen";
+                }
```

Comment:
In the lab, when running automation, we call `.\tests\legacyjit_x86_testenv.cmd`. It sets these variables: COMPLUS_AltJit, COMPLUS_AltJitNgen, COMPLUS_AltJitName, COMPLUS_NoGuiOnAssert, COMPLUS_AltJitAssertOnNYI. Have you tested this so they do not step into "this" parent process inherited environment? What permutation will we be running on the lab?

Reply:
Not quite sure what you are asking -- is it about the impact of these variables on the parent harness process or on the child music bench process or both?

Normally we will execute the test harness using the just-built corerun. Any environment variables set when launching corerun will impact the execution of the harness, and will get passed down to any child processes.

So setting these COMPlus vars before running the harness will impact both parent and child. This generally should not cause problems.

The environment updates done inside the harness only apply to the child music bench process. These updates might in override settings in the ambient environment, but that seems entirely reasonable.

Reply:
Just wonder if setting the environment before running the harness might make these settings obscure. What is the user actually running? Is this meant for local runs only?

Reply:
We can't control what COMPlus settings are there when the harness is invoked. We could enmerate or perhaps filter the setttings that get passed down from the harness to the child music store; that might be useful, but I don't see it as super-important.

Reply:
@AndyAyersMS are we planning on increasing the permutation of what gets running in the lab? If so, what are the new permutations? I am asking because at the moment the permutations are controlled from the `perf.groovy` which invokes the `run-xunit-perf` script.